backend/core/src/units/dto/unit.dto.ts

```diff
@@ -15,6 +22,27 @@ export class UnitDto extends OmitType(Unit, ["property", "amiChart"] as const) {
   @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
   @Type(() => AmiChartDto)
   amiChart: AmiChartDto | null
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => IdDto)
+  unitTypeRef?: IdDto
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => IdDto)
+  unitRentType?: IdDto
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => IdDto)
+  unitAccessibilityPriorityType?: IdDto
```

Comment:
I think we should be consistent in the naming conventions for these reference IDs. This convention doesn't seem to be followed anywhere, but it makes sense to me to name these fields based off of what they are, the id, so it would be unitTypeId, for example.